src/features/user/TreeMapper/Analytics/components/TreePlanted/index.tsx

```diff
@@ -530,11 +530,7 @@ export const TreePlanted = () => {
 
     const isValidTimeFrame =
       timeFrame && getTimeFrames(toDate, fromDate).includes(timeFrame);
-    if (
-      process.env.ENABLE_ANALYTICS === 'true' &&
-      isValidTimeFrame &&
-      project
-    ) {
+    if (process.env.ENABLE_ANALYTICS && isValidTimeFrame && project) {
       fetchPlantedTrees();
```

Comment:
I feel we should explicitly check ENABLE_ANALYTICS for a specific value and otherwise disable it. In the current implementation, any value will enable analytics.

Reply:
@Shreyaschorge

Reply:
Not applicable as ENABLE_ANALYTICS is a boolean value coming from next.config.js